### ISSUE TITLE
Adjust recommended JVM Memory settings

### ISSUE THREAD

**Issue by vanEgmond** [geerlingguy/ansible-role-solr#133]
Currently the README recommends:

> solr_xms: "256M"
solr_xmx: "512M"

> Memory settings for the JVM. These should be set as high as you can allow for best performance and to reduce the chance of Solr restarting itself due to OOM situations.

However as per the Solr reference documentation: https://solr.apache.org/guide/8_11/jvm-settings.html

> Lucene/Solr makes extensive use of MMapDirectory, which uses RAM not reserved for the JVM for most of the Lucene index. Therefore, as much memory as possible should be left for the operating system to use for this purpose.

Ensuring that the systems have enough RAM which is not allocated to the JVM heap is paramount for performance, and should be around 25-50% of total RAM. Not to be confused with the recommended 25-50% headroom over the minimum tested JVM Heap usage mentioned in the documentation, specific per setup/use-case.

**Comment by geerlingguy** [comment]
Yeah, I think we could update those suggestions (and maybe bump the defaults to at least 512/1G) since when I wrote this originally, I think it was on Solr 1.4 or so :D
